client/store/modules/auth/actions.js

```diff
@@ -2,14 +2,14 @@ import api from '@/api/auth';
 
 export const login = ({ commit }, credentials) => {
   return api.login(credentials)
-    .then(user => commit('login', user));
+    .then(data => commit('login', data));
```

Comment:
I would prefer destructured data (if we don't have naming for user + token) to see what is being passed as data. One other possible solution would be to pass token within user object.